lib/util/index.js

```diff
@@ -520,6 +520,40 @@ function valueToJSON(value) {
   return value;
 }
 
+/**
+ * Create the bandwidth profile payload included in an RSP connect message.
+ * @param {BandwidthProfileOptions} bandwidthProfile
+ * @returns {object}
+ */
+function createBandwidthProfilePayload(bandwidthProfile) {
+  const payload = {};
+  if (typeof bandwidthProfile.video !== 'object') {
+    return payload;
+  }
+
+  const mode = bandwidthProfile.video.mode;
+  if (typeof mode === 'string') {
+    payload.mode = mode;
```

Comment:
do we need extra validation on valid values `mode ` for this ? I am okay with not having more validations, but just wondering.

Reply:
@makarandp0 I don't think so. We can add validation once we have the latest RSP spec implemented by Room Service.